server/src/main/java/com/genymobile/scrcpy/Workarounds.java

```diff
@@ -154,13 +167,29 @@ private static void fillAppContext() {
         }
     }
 
+    @SuppressLint("PrivateApi")
+    public static Context getSystemContext() throws ReflectiveOperationException {
+        if (systemContext == null) {
+            try {
+                // Hide warnings on XiaoMi devices
```

Comment:
I'd like to include this to reduce noise on XiaoMi devices, and test it on other devices to make sure it doesn't cause any side effects.